cookbooks/aws-parallelcluster-environment/recipes/init/restore_home_shared_data.rb

```diff
@@ -18,19 +18,65 @@
   # Restore the shared storage home data if it doesn't already exist
   # This is necessary to preserve any data in these directories that was
   # generated during the build of ParallelCluster AMIs after converting to
```

Comment:
Why during the Pcluster AMI build? This recipe is executed at node bootstrap time.

Reply:
I didn't modify it, it was like this previously. But I agree with you. Should we modify this comments?

Reply:
Yes, but not a blocker for the PR

Reply:
Just understand the previous comment. It's saying the data generated during the build of ParallelCluster AMIs. Not this recipe is executed during the build of ParallelCluster AMIs.

Reply:
Modified the comment to the original one.